checkstyle.xml

```diff
@@ -107,11 +108,11 @@
 		</module>
 
 		<module name="Indentation">
-			<property name="basicOffset" value="8"/>
+			<property name="basicOffset" value="4"/>
 			<property name="caseIndent" value="0"/>
```

Comment:
Hmm, @sfPlayer1 can I request to change this into 4 (1 tab)? It looks weird currently, like https://github.com/FabricMC/MappingPoet/blob/8dc2fd9864a2085784090ea5a9423295e64bd6c3/src/main/java/net/fabricmc/mappingpoet/Signatures.java#L210-L232

Reply:
Will keep this.